lib/Console/Application.php

```diff
@@ -122,6 +122,10 @@ protected function getDefaultInputDefinition(): InputDefinition
 
     public function add(Command $command): ?Command
     {
+        if ($command instanceof LazyCommand) {
+            $command = $command->getCommand();
+        }
```

Comment:
Does this make all command non-lazy?

Reply:
Yes, this makes all doctrine commands non-lazy which shouldn't have have any impacts. This is the only solution we have at the moment to fix running the commands again.

Reply:
Also when calling `list` this initiates all commands including lazy ones so i would go with this fix.

Reply:
They haven't been lazy until `doctrine/migrations` `v3.6.1` anyway...